SRM release notes for release 803 (#2949)
harness/developer-hub: release-notes/service-reliability-management.md

Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page. 
:::

## Latest: Version 803xx

### New features and enhancements

This release does not include new features.

### Early access features
This release does not include any early access features.

### Fixed issues

> - Users were unable to update the change source configuration within their monitored service.(SRM-15083)

Reviewer: Unless the audience for this note is not the user who experienced this issue, can we write in the second person? Retain if the reader is, say, an account admin who reported the issue on behalf of their users.

> - Users were unable to update the change source configuration within their monitored service.(SRM-15083)

Author: ```suggestion
- Unable to update the change source configuration within their monitored service. (SRM-15083)
```

>   This issue has been resolved. Users can now update the change source configurations in their monitored service.

Reviewer: Plural "configurations"? The issue description has the singular. Even if it's possible to have more than one change source config, I just thought I'll let you know that the difference stood out to me.

>   This issue has been resolved. Users can now update the change source configurations in their monitored service.

Author: ```suggestion
  This issue has been resolved. You can now easily update the change source configurations within your monitored service.
```
